.circleci/config.yml

```diff
@@ -258,6 +258,52 @@ jobs:
               run --rm --no-deps app \
                 black src/richie/apps src/richie/plugins sandbox --check
 
+  extract-back-strings:
```

Comment:
we usualy add docstrings on top of the jobs with all the insight about what you've done and why you've done it like this.

Is there a reason why you didn't do it in the back's "build" job like you did for the front?

Reply:
As far as I can see there is no backend "build" job

Reply:
There are 3 of them: build, build-dev and build-alpine.

I agree that the names are not coherent. We should probably rename them to: build-back-debian, build-back-debian-dev and build-back-alpine...

Reply:
:+1: